### DISCUSSION TITLE
Unit testing react error: "t.findViewByModel is not a function"

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1875] [Q&A]
Hello,

Can somebody help in test with jest in reactJS. I would like to test a drawing line but I have this error: 

<img width="170" alt="Capture d’écran 2022-11-03 093043" src="https://user-images.githubusercontent.com/83480447/199676518-126eff5b-f637-4a93-a883-80a5b05670dd.png">

And, here is my code:

![code](https://user-images.githubusercontent.com/83480447/199676667-83ef42d3-4dfa-45a3-8f28-51c377812e0f.png)

I've tried everything to mock the function and nothing to do.
I don't understand

Thanks fo your help

**Comment by kumilingus** [comment]
It seems like the `paper` is not an instance of `joint.dia.Paper`.
